pkg/controller/utils/dnszone.go

```diff
@@ -0,0 +1,101 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
```

Comment:
the grouping of the imports seems inconsistent. only one controller-runtime import is by itself.